---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

Education
======
* <b>D.Sc., Animal Science and Pastures</b>, University of São Paulo - Piracicaba.
  * Dissertation: "Multi-omic data integration for the study of production, carcass and meat quality traits in Nellore cattle".
  * Advisor: Dr. Luiz Lehmann Coutinho.
  
* <b>M.Sc., Animal Biocience</b>, University of São Paulo-Pirassununga
  * Thesis: "Analysis in serum metabolomic associated with feed efficiency in Nellore".
  * Advisor: Dr. Heidge Fukumasu.
  
* <b>B.S., Animal Science</b>, University of São Paulo-Pirassununga
  * Advisor: Dr. Heidge Fukumasu.

Work experience
======
<b>August 2022- Now </b>: Postdoctoral fellow at University of Alberta
  * Faculty of Agricultural, Life & Environmental Sciences
  * Supervisor: Edward Bork, Carolyn Fitzsimmons
  * Postodoc in Data Science and Precision Livestock Farming

<b>September 2021- August 2022</b>: Postdoctoral fellow at University of São Paulo
  * Department of Medicine Veterinary, FZEA/USP
  * Supervisor: Heidge Fukumasu
  * Postodoc in Machine-learning applied to Metagenomics

<b>August 2017 - September 2021</b>: Graduate Research Assistant (D.Sc.)
  * Department of Animal Science, ESALQ/USP
  * Advisor: Luiz Lehmann Coutinho
  * Grant number:	88882.328657/2019-01 (CAPES)
  * My P.h.D research has focused on Multi-omics Data integration methods applied to meat quality in Nellore cattle.

* <b>August 2019 - February 2020</b>: Exchange Visitor 
  * Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg,VA.
  * Supervisor: Gota Morota
  * Grant number:	88887.367967/2019-00 (CAPES)
  * During the six-months internship I was training on machine-learning approaches applied to data omics integration.
  
* <b>Ouctober 2016- April 2017</b>: Exchange Visitor
  * School of Chemical & Biomolecular Engineering, Georgia Institute of Technology, Atlanta,GA.
  * Supervisor: Mark Styczynski
  * Grant number:	16/16214-8 (FAPESP)
  *  During the six-months internship I was training on preprocessing, univarite and multivariate analysis for metabolomics.
 
* <b>Ouctober 2014- June 2017</b>: Graduate Research Assistant (M.Sc.)
  * Department of Medicine Veterinary, FZEA/USP
  * Advisor: Heidge Fukumasu
  * Grant number:	15/01059-4 (FAPESP)
  *  My research was focused to identify potential biomarkers associated with feed efficiency as well as to understand the molecular process that explains feed efficiency variation.
  
* <b>August 2013- June 2014</b>: Technical Assistant
  * CRV Lagoa Ltda, Sertãozinho, São Paulo
  * Dairy Cattle Animal Breeding Program
  * At CRV Lagoa he worked with the database of animal breeding program for dairy cattle, he also supported dairy cattle mating via CRV ’s mating program.
  
* <b>December 2012- June 2013</b>: Undergraduate Research Assistant
  * Department of Basic Science, FZEA/USP
  * Grant number:	12/17153-1 (FAPESP)
  * Advisor: Heidge Fukumasu
  * Project:"Standardization of Genome Wide Association Study (GWAS) based on DNA pooling and allelotyping in high density arrays in Nellore"
  * The aim of this work is standardize an experimental model for phase one GWAS based on DNA pooling from animals with high and low efficiency for age at first calving simultaneously with higher body weight gain of calves.
  
* <b>June 2011- June 2012</b>: Undergraduate Research Assistant
  * Department of Basic Science, FZEA/USP
  * Grant:	CNPq
  * Advisor: José Bento Sterman Ferraz
  * Project:"Genetic and Phenotypic correlations between rump eye area and backfat thickness mesuared in different anatomics points of <i>Longissimus Dorsi</i> of Nellore cattle"
  
Skills
======
* ### Quantitative Skills
  * <b>Transcriptome Analysis</b>
    * RNA-sequencing: DEseq2, NOISeq, EdgeR, LIMMA
    * Co-expression network analysis: Weighted Gene Coexpression Network Analysis (WGCNA), Cytoscape
  
  * <b>Metagenomics Analysis</b>
    * Processing and Phylogenetics: Fastqc. Multiqc, Figaro, HTStream, DADA2, Phyloseq, Vegan
    * Statistical Analysis: Phyloseq, DEseq2, EdgeR


  * <b>Metabolomics Analysis</b>
    * Preprocessing: Progenesis QI Software, XCMS 
    * Statistical Analysis: MetaboAnalyst
    * Putative Identification: mummichog
  * <b>Statistics</b>
    * R
* ### Computational Skills
   * <b>Operating Systems: LINUX, Windows</b>
     * R skills: LATEX, ggplot2, Tidyr, dplyr, stringr, purrr, tibble.
      * Photoshop and profissional slides.
  
Publications
======
  <ul>{% for post in site.publications %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
  
Talks
======
  <ul>{% for post in site.talks %}
    {% include archive-single-talk-cv.html %}
  {% endfor %}</ul>
  
Teaching
======
  <ul>{% for post in site.teaching %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
